guides/common/modules/proc_deleting-a-vm-on-amazon-ec2.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Deleting_a_VM_on_Amazon_EC2_{context}"]
+= Deleting a VM on Amazon EC2
+
+You can delete VMs running on Amazon EC2 from within {Project}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Infrastructure > Compute Resources*.
+. Select your Amazon EC2 provider.
+. On the *Virtual Machines* tab, click *Delete* from the *Actions* menu.
+This deletes the virtual machine from the Amazon EC2 compute resource while retaining any associated hosts within {Project}.
+If you want to delete the orphaned host, navigate to *Hosts > All Hosts* and delete the host manually.
```

Comment:
```suggestion
If you want to delete an orphaned host, navigate to *Hosts* > *All Hosts* and delete the host manually.
```
Is this orphaned host specific or could there be multiple instances?

Reply:
no, could be multiple. Applied.